iis-workflows/iis-workflows-metadataextraction/src/test/resources/eu/dnetlib/iis/workflows/metadataextraction/sampledataproducer/data/extr_meta.json

```diff
@@ -499,5 +503,5 @@
   "issue": null,
   "pages": null,
   "publicationTypeName": null,
-  "text": "This is sample PDF document.\nGood man, I\u0027m a good man, food\u0027s on the table, working two jobs,\nWaiting willing the neighbors check your name\nI\u0027m having fun, got no kids and I love the Lord, check your name\nI\u0027m monogamous never did time but maybe just once, check your name\nAnd I puts it down, wanna see it twice, brother puts it down, check\nHow could you ever walk away, after all I\u0027ve done for you\nI feel like there\u0027s a knife in my back, babe\nYou might as well pull it all the way through\nGood man, I\u0027m a good man, food\u0027s on the table, working two jobs,\nWaiting willing the neighbors check your name, good man\nI\u0027m having fun, got no kids and I love the Lord, check your name\nI\u0027m monogamous never did time but maybe just once, check your name\nAnd I puts it down, wanna see it twice, brother puts it down, check\nYou watched them hand cuff me lady, and walk me to my room\nI never been so humiliated, I don\u0027t think that I deserve it for you\nGood man, I\u0027m a good man, food\u0027s on the table, working two jobs,\nWaiting willing the neighbors check your name, good man\nI\u0027m having fun, got no kids and I love the Lord, check your name\nI\u0027m a monogamus never did time but maybe just once, check your name\nAnd I puts it down, wanna see it twice, brother puts it down, check\nI never been, I never been so much pain, never\nMy life is better, my love is pretty, so much better, now\nWithout you, I never had so ... without you, whenever ... inside\nHow could you... just ...walk away\nWhen the things go rough you quickly ran into .. somebody say, I\u0027ve been there\nThere\u0027s one thing I know so I\u0027ma tell you that I\u0027m a good man, back to you, a good man\nGood man, good man, good man, good man."
+  "text": ""
```

Comment:
@dtkaczyk it seem cermine was unable to extract text after upgrading dependency to 1.8-SNAPSHOT. Was it expected?